src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -22,10 +24,12 @@
 public class UserTenantServiceImpl implements UserTenantService {
```

Comment:
Please add class-level java doc to explain why we not storing consortiumId in user_tenant table and so why not joining to this consortiumId.
Our consortium table will contain only single record and it will be prohibited to add another record in this table.
If other consortium will be created - the new separate DB schema for it will be created that also store only single record in consortium table.

So to simplify logic and source code it was decided that we will not store consortiumId in user_tenant table

Reply:
done